pkg/cmd/adm/e2ed/controller.go

```diff
@@ -0,0 +1,219 @@
+package e2ed
```

Comment:
TODO: move the controller to a dedicated package structure (removing from command package)+